resources/rafter/charts/controller-manager/charts/minio/values.yaml

```diff
@@ -13,8 +13,8 @@ image:
 ## client used to create a default bucket).
 ##
 mcImage:
-  repository: minio/mc
-  tag: edge
+  repository: eu.gcr.io/kyma-project/incubator/develop/minio/mc
```

Comment:
Please add this to `resources/rafter/values.yaml` under `controller-manager.minio` path :)